### PR TITLE
Rename Blacklisted Apps to Blocked Apps

### PR DESCRIPTION
**Pull request by SyntaxNode**
Replaces all wording of "blacklisted apps" to '"blocked apps". This is a breaking change for host configurations and metrics. There is no change to the API.

### CODE REVIEW

File: metrics/config/metrics_test.go [hunk 1]
```diff
@@ -150,14 +150,14 @@ func TestMultiMetricsEngine(t *testing.T) {
...
 	VerifyMetrics(t, "RequestStatuses.Video.BadInput", goEngine.RequestStatuses[metrics.ReqTypeVideo][metrics.RequestStatusBadInput].Count(), 0)
-	VerifyMetrics(t, "RequestStatuses.Video.BlacklistedAcctOrApp", goEngine.RequestStatuses[metrics.ReqTypeVideo][metrics.RequestStatusBlacklisted].Count(), 1)
+	VerifyMetrics(t, "RequestStatuses.Video.BlockededAcctOrApp", goEngine.RequestStatuses[metrics.ReqTypeVideo][metrics.RequestStatusBlockedApp].Count(), 1)
 	VerifyMetrics(t, "RequestStatuses.OpenRTB2.Error", goEngine.RequestStatuses[metrics.ReqTypeORTB2Web][metrics.RequestStatusErr].Count(), 0)
 	VerifyMetrics(t, "RequestStatuses.OpenRTB2.BadInput", goEngine.RequestStatuses[metrics.ReqTypeORTB2Web][metrics.RequestStatusBadInput].Count(), 0)
-	VerifyMetrics(t, "RequestStatuses.OpenRTB2.BlacklistedAcctOrApp", goEngine.RequestStatuses[metrics.ReqTypeORTB2Web][metrics.RequestStatusBlacklisted].Count(), 0)
+	VerifyMetrics(t, "RequestStatuses.OpenRTB2.BlockedAcctOrApp", goEngine.RequestStatuses[metrics.ReqTypeORTB2Web][metrics.RequestStatusBlockedApp].Count(), 0)
```
Comment (bsardo) [inline metrics/config/metrics_test.go:160]: Perhaps these three test metrics should not refer to `Acct`:
`RequestStatuses.AMP.BlockedApp`
`RequestStatuses.Video.BlockededApp`
`RequestStatuses.OpenRTB2.BlockedApp`

File: config/config.go [hunk 2]
```diff
@@ -1138,8 +1138,7 @@ func SetupViper(v *viper.Viper, filename string, bidderInfos BidderInfos) {
 	v.SetDefault("default_request.type", "")
 	v.SetDefault("default_request.file.name", "")
 	v.SetDefault("default_request.alias_info", false)
-	v.SetDefault("blacklisted_apps", []string{""})
-	v.SetDefault("blacklisted_accts", []string{""})
```
Comment (bsardo) [inline config/config.go:1142]: It looks like the concept of blacklisted accounts was removed a while ago but this `SetDefault` call was missed, is that correct?

File: metrics/config/metrics_test.go (hunk 1 above)
Comment (SyntaxNode) [reply metrics/config/metrics_test.go:160]: It's just the test name. Fixed.

File: config/config.go (hunk 2 above)
Comment (SyntaxNode) [reply config/config.go:1142]: Correct.

**Review by bsardo — APPROVED** [review]

**Review by VeronikaSolovei9 — APPROVED** [review]

**Pull request merged**